aiven/client/cli.py

```diff
@@ -3059,6 +3059,86 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List created Flink tables"""
+        project_name = self.get_project()
+        self.print_response(self.client.list_flink_tables(project_name, self.args.service_name))
+
+    @arg.project
+    @arg.service_name
+    @arg("integration_id", help="Service integration ID")
+    @arg("-n", "--table_name", required=True, help="Table name")
+    @arg("-kt", "--kafka_topic", required=False, help="Topic name, used as a source/sink. (Kafka integration only)")
+    @arg("-dt", "--jdbc_table", required=False, help="Table name in Database, used as a source/sink. (PG integration only)")
+    @arg("-p", "--partitioned_by", required=False, help="A column from a schema, table will be partitioned by")
+    @arg(
+        "-l",
+        "--like_options",
+        required=False,
+        help="Clause can be used to create a table based on a definition of an existing table"
+    )
+    @arg("-s", "--schema_sql", required=True, help="Source/Sink table schema")
+    def service__flink__table__create(self):
+        """Create a Flink table"""
+        project_name = self.get_project()
+        self.print_response(
+            self.client.create_flink_table(
+                project_name,
+                self.args.service_name,
+                self.args.integration_id,
+                self.args.table_name,
+                self.args.schema_sql,
+                self.args.kafka_topic,
+                self.args.jdbc_table,
+                self.args.partitioned_by,
+                self.args.like_options,
+            )
+        )
+
+    @arg.project
+    @arg.service_name
+    @arg("table_id", help="Table ID")
+    def service__flink__table__get(self):
+        """Get a Flink table"""
+        project_name = self.get_project()
+        self.print_response(self.client.get_flink_table(
+            project_name,
+            self.args.service_name,
+            self.args.table_id,
+        ))
+
+    @arg.project
+    @arg.service_name
+    @arg("table_id", help="Table ID")
+    def service__flink__table__delete(self):
+        """Delete a Flink table"""
+        project_name = self.get_project()
+        self.print_response(self.client.delete_flink_table(
+            project_name,
+            self.args.service_name,
+            self.args.table_id,
+        ))
+
+    @arg.project
+    @arg.service_name
+    @arg("job_name", help="Job Name")
+    @arg("-t", "--tables", required=False, help="List of tables required in job runtime")
```

Comment:
This could have example or something to say this is comma separated list?
```suggestion
    @arg("-t", "--tables", required=False, help="List of tables required in job runtime, e.g. table1,table2,table3")
```

Reply:
Instead of parsing manually from comma separated list using `nargs="*"` would be better